mixin/build.go

```diff
@@ -161,7 +161,15 @@ func buildMix(prepNeeded bool) error {
 		_ = os.Remove(mixFlagFile)
 		return err
 	}
-	err = b.BuildUpdate("", 0, "", false, true, false)
+	params := builder.UpdateParameters{
+		MinVersion:    0,
+		Format:        "",
+		Publish:       true,
+		SkipSigning:   false,
+		SkipFullfiles: false,
+		SkipPacks:     false,
+	}
+	err = b.BuildUpdate(params)
```

Comment:
maybe
 ``` err := b.BuildUpdate(&builder.UpdateParameters{Publish: true})```
to use a literal struct, as there is just one field that is taking the non-default value in this case.

Reply:
ah nice! I didn't know that language feature.